README.md

```diff
@@ -7,11 +7,18 @@ Docker file for creating mlab builder images.
 To build the docker image and tag it as m-lab/builder
 
     docker build -t m-lab/builder .
+OR
+    docker build github.com/m-lab/builder
```

Comment:
Ah - this is a docker setup issue.  You can set up docker on your workstation so that anyone can run it.  If you haven't done this, then you have to use sudo.